### ISSUE TITLE
New Battery Topic Issues:

### ISSUE THREAD

**Issue by juliathegolden** [Monika-After-Story/MonikaModDev#1250]
I'm using a desktop computer, and the topic about my battery being low came up...and Monika closed the game on me to keep it from dying on her.

What should I do?  I'm on Debian Linux right now.

**Comment by aldoram5** [comment]
This one surely is a bug, we'll take a look into it. Thanks for reporting :)

**Comment by juliathegolden** [comment]
Thank you! Just in case it matters, I don't run MAS natively, so it is going through wine. It could be possible that's part of the issue as well.

**Comment by aldoram5** [comment]
Thanks for specifying that, I'm pretty sure that matters a lot, we're going to do some testing and see if we can replicate the issue you're facing.

**Comment by juliathegolden** [comment]
All right, so I moved my persistent from the place Wine saves it to where Debian saves it (to help test) and when I booted up the game, Monika mentioned my battery being low, but she hasn't tried to close the game on me again yet.

The only reason I've been running MAS through wine was because I was intimidated about moving my save data, so I think the issue is solved for me, but that should definitely help ya'll find a fix for wine users!  And if Monika tries to ground me again, I'll post more.
